logger/logger.go

```diff
@@ -230,27 +237,23 @@ func newCustomResponseWriter(w http.ResponseWriter) *customResponseWriter {
 	}
 }
 
-// WriteHeader implements ResponseWriter and saves status
 func (c *customResponseWriter) WriteHeader(status int) {
```

Comment:
I think lack of the comment on exported func (even on non-exported struct) may cause complains with some linters and IDEs